src/coreComponents/mesh/WellElementRegion.hpp

```diff
@@ -121,17 +121,17 @@ class WellElementRegion : public ElementRegionBase
   /**
    * @brief Not implemented, this task is performed in GenerateWell.
    */
-  virtual void generateMesh( Group & ) override {}
```

Comment:
Another thing if I am not wrong , the rule is:  Use override (only) for a derived class' override to help maintenance. So virtual isn't necessary here.

Reply:
Changed it